src/modules/mount/main.py

```diff
@@ -321,6 +327,12 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap']
+    swap_device="".join(map(str,swap_partitions))
```

Comment:
I don't think the call to `map()` is needed but more importantly this will concat all the devices together in a string with no separator.  I don't think that is what you want here.